modules/tinymce/src/core/main/ts/api/EditorCommands.ts

```diff
@@ -407,6 +408,10 @@ class EditorCommands {
         LineHeightCommands.lineHeightAction(editor, value);
       },
 
+      'Lang': (command, ui, value) => {
+        ContentLanguageCommands.contentLanguageAction(editor, value);
```

Comment:
As I mentioned the other day, this should be using the existing the existing `toggleFormat` function like `Bold`, etc... as well. So this should be like this:
```suggestion
        self.toggleFormat(command, {
          value: value.code,
          customValue: value.customCode
        });
```

Reply:
I should add I'm guessing this may have been done since you went to do the query command and were trying to keep it the same as LineHeight and FontSize. However, since we're not doing the query command anymore it should be consistent with the others.